ipfs-cluster-service/main.go

```diff
@@ -210,15 +210,14 @@ configuration.
 					Name:  "custom-secret, s",
 					Usage: "prompt for the cluster secret",
 				},
-				cli.StringFlag{
-					Name:   "env-cluster-secret",
-					EnvVar: "CLUSTER_SECRET",
-					Hidden: true,
-				},
 			},
 			Action: func(c *cli.Context) error {
-				initConfig(c.GlobalBool("force"), !c.Bool("custom-secret"),
-					c.String("env-cluster-secret"))
+				envSecret, envSecretDefined := os.LookupEnv("CLUSTER_SECRET")
+				if envSecretDefined {
```

Comment:
Nice, yeah I like that because it cleans up the logic around `initConfig()` and takes care of grabbing a user-specified secret in one place (instead of doing part of it outside of `initConfig()` and part of it inside).